common-npm-packages/packaging-common/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "packaging-common",
```

Comment:
I would propose to change it to `azure-pipelines-tasks-packaging-common`.

Reply:
Updated the change.